cli/pcluster/cfnconfig.py

```diff
@@ -530,6 +533,23 @@ def __init_cluster_parameters(self):
             except configparser.NoOptionError:
                 pass
 
+    def __init_efa_parameters(self):
+        try:
+            __temp__ = self.__config.get(self.__cluster_section, "enable_efa")
+            if __temp__ != "compute":
+                self.__fail("valid values for enable_efa = compute")
+
+            supported_features = get_supported_features(self.region, "efa")
+            valid_oses = supported_features.get("baseos")
+            valid_schedulers = supported_features.get("baseos")
```

Comment:
Forgot to replace “baseos” with schedulers

Reply:
done, also I've removed the fetching the json from this portion to just do hardcoded validation (for now) as discussed.